test/unit/controllers/ileQuery/IleQueryControllerSpec.scala

```diff
@@ -369,7 +369,7 @@ class IleQueryControllerSpec extends ControllerLayerSpec with MockIleQueryCache
             when(ileQueryRepository.findBySessionIdAndUcr(anyString(), anyString()))
               .thenReturn(Future.successful(Some(IleQuery("sessionId", "mucr", conversationId))))
             when(ileQueryRepository.removeByConversationId(anyString())).thenReturn(Future.successful((): Unit))
-            val backendResponse = HttpResponse(OK, Some(Json.toJson(Seq(ucrNotFoundResponseExchange))))
+            val backendResponse = HttpResponse(status = OK, json = Json.toJson(Seq(ucrNotFoundResponseExchange)), headers = Map.empty)
```

Comment:
Due to deprecated apply method in HttpResponse we need to specify names of the arguments manually, because deprecated apply have the same type arguments